uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/ui/widgets/WidgetBehavior.kt

```diff
@@ -37,40 +37,50 @@ class WidgetBehavior @Inject constructor(
 ) {
     fun onAddRepetition(habit: Habit, timestamp: Timestamp?) {
         notificationTray.cancel(habit)
-        setValue(habit, timestamp, Entry.YES_MANUAL)
+        val entry = habit.originalEntries.get(timestamp!!)
+        val notes = entry.notes
+        setValue(habit, timestamp, Entry.YES_MANUAL, notes)
```

Comment:
The `notes` variable seems redundant here. The last line could be written as `setValue(habit, timestamp, Entry.YES_MANUAL, entry.notes)`. Same comment for the changes below this one.